docs/zkapps/lightnet.mdx

```diff
@@ -0,0 +1,214 @@
+- Prepares the file system
+- Downloads and updates the lightweight Mina Explorer
+- Forms the network with a single node
+- Configures the network to be fast
+- Disables the blockchain SNARK proofs
+- Uses the `o1js-main` branch of the Mina repository
+- Starts the Mina archive process and the archive node API
+- Waits for the network to reach a synchronized state
+- Sets the Mina process logging level to trace
+
+## Start a local network with other settings
+
+To see the options to start a local blockchain network with non-default settings:
+
+```sh
+zk lightnet start --help
+```
+
+You can configure different network properties. For example, to start a multi-node local blockchain network with slower, close-to-real-world properties:
+
+```sh
+zk lightnet start --mode multi-node --type real
+```
+
+## Keep current versions
+
+New Docker images are built and published to the Docker Hub every night. You might not always want to get the latest product versions. For example, when your zkApp relies on well-defined APIs and you want to continue developing in your current environment. 
+
+To keep your current working versions of o1js and Mina:
+
+```sh
+zk lightnet start --no-pull
+```
+
+## Stop the local network
+
+To stop the local network, remove the Docker container, and clean up the environment:
+
+```sh
+zk lightnet stop
+```
+
+When Lightnet is stopped, the log files for Docker container services are saved to the host file system at `/Users/<username>/.cache/zkapp-cli/lightnet/logs/`.
+
+To disable saving of log files:
+
+```sh
+zk lightnet stop --no-save-logs
+```
+
+## Troubleshooting transactions
+
+Tools are provided for troubleshooting.
+
+### Lightweight Mina Explorer
+
+To monitor transactions, launch the lightweight Mina Explorer:
+
+```sh
+zk lightnet explorer
+```
+
+By default, the latest version is launched. 
+
+To list versions, their published dates, and show the version in use:
+
+```sh
+zk lightnet explorer list true
+```
+
+To use a specific version of the lightweight Mina Explorer:
+
+```sh
+zk lightnet explorer use <version>
+```
+
+### Reinstantiate lightnet
+
+
+
+### Log files
+
+Log files for different processes are saved inside the Docker container as:
+
+- `/root/logs/*.log`
+- `/root/.mina-network/mina-local-network-2-1-1/nodes/**/logs/*.log`
+
+To save the log files that are produced by Docker container processes to the host machine file system:
+
+```sh
+zk lightnet logs save
+```
+
+To debug your zkApp, you can follow the Docker container process logs in real time. 
+
+On a different terminal window, to stream the process logs:
+
+```sh
+zk lightnet logs follow
+```
+
+Select the Docker container process to follow. Press Ctrl+C to stop streaming.
+
+
+## Lightnet status
+
+To get the network status:
+
+```sh
+zk lightnet status
+```
+
+The network status is returned, including useful URLs. For example, the default single node network status returns these useful URLs:
```

Comment:
```suggestion
The network status is returned, including useful URLs. For example, the default single node network status returns these endpoints:
```
??

Reply:
Your question marks mean you have a question? Can you elaborate please? @nicc